src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -338,5 +338,5 @@ class ChannelController implements ITelemetryPlugin {
 }
 
 const validationError = "Extensions must provide callback to initialize";
-const ChannelControllerPriority = 200;
+const ChannelControllerPriority = 999;
```

Comment:
Who gets priority 1000?

Reply:
No, one, I updated this to move channelcontroller to 500. Its not arbitary, I realize we have scenarios where customers need to udpate data stream before channel and need two channels. So, we need prioritization in each parallel channel pipeline which is a supported scenario.